tools/fapi/tss2_verifyquote.c

```diff
@@ -100,7 +100,7 @@ int tss2_tool_onrun (FAPI_CONTEXT *fctx) {
     if (ctx.signature) {
         r = open_read_and_close (ctx.signature, (void**)&signature, &signatureSize);
         if (r) {
-            free (qualifyingData);
+            if (qualifyingData) free (qualifyingData);
```

Comment:
no if, here and throughout on raw free() calls...

Reply:
Fixed in updated PR.